src/core/projectTeams/test/projectTeam.service.spec.ts

```
@@ -1,1024 +1,1335 @@
+                await expect(
+                    service.updateProjectTeam(
+                        projectId,
+                        userId,
+                        mockUpdateProjectTeamRequest,
+                        mockFiles,
+                    ),
+                ).rejects.toThrow(ProjectTeamMainImageException);
+            });
+            it('업데이트 되는 deleteMainImage가 2개 이상이면 ProjectTeamMainImage 예외가 발생한다.', async () => {
+                const request = mockUpdateProjectTeamRequest;
+                request.deleteMainImages = [1, 2];
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)